cli/src/tasks/add.ts

```diff
@@ -93,16 +94,18 @@ export async function generateCapacitorConfig(config: Config) {
     return;
   }
 
-  const inquirer = await import('inquirer');
-  const answers = await inquirer.prompt([
-    {
-      type: 'input',
-      name: 'webDir',
-      message:
-        'What directory are your web assets in? (index.html, built JavaScript, etc.):',
-      default: 'www',
-    },
-  ]);
+  const answers = await prompts(
+    [
+      {
+        type: 'text',
+        name: 'webDir',
+        message:
+          'What directory are your web assets in? (index.html, built JavaScript, etc.):',
+        initial: 'www',
+      },
+    ],
+    { onCancel: () => process.exit(1) },
+  );
```

Comment:
Looks like this code is unreachable, the whole `generateCapacitorConfig` could be deleted. The `capacitor.config.json` is generated on init, so it always exists on add. And if it doesn't exist add won't work anyway, so it never prompts for webDir.
Maybe it should be moved to the init?

Reply:
Yeah, I can do that as part of https://github.com/ionic-team/capacitor/issues/3465.

This PR isn't trying to change behavior, just replace the prompt library

Reply:
yeah, but I think a better way of replacing a prompt that is not used is deleting it 😛